crt_portal/cts_forms/templates/forms/complaint_view/print/form_letter.html

```diff
@@ -29,6 +29,10 @@ <h2 class="form-letterhead--header-subtext">Civil Rights Division</h2>
       {% endif %}
       <br />
       {% if data.contact_city %}{{ data.contact_city }},{% endif %} {% if data.contact_state %}{{ data.contact_state }}{% endif %} {% if data.contact_zip %}{{ data.contact_zip }}{% endif %}
+      {% if data.contact_email %}
```

Comment:
Should `data.contact_email` also be added to the list of conditionals in line 19?